src/charactersheet/viewmodels/common/root/index.html

```diff
@@ -209,7 +209,7 @@ <h4 class="modal-title" data-dismiss="modal">Create a new character or campaign<
     </ul>
     <center>
       <small class="text-muted">
-      &copy; 2016-2017 Adventurer's Codex, LLC. All rights reserved.<br />
+      &copy; 2016-2018 Adventurer's Codex, LLC. All rights reserved.<br />
```

Comment:
This should be calculated. We should never have to go in here and update this field year after year. I would suggest looking at how the build date is calculated and do something similar.